mat/dense_test.go

```diff
@@ -2093,3 +2093,14 @@ func denseMulTransSymBench(b *testing.B, size int, rho float64) {
 		wd = &n
 	}
 }
+
+func BenchmarkDenseSum1000(b *testing.B) { denseSumBench(b, 1000, 100) }
```

Comment:
This looks like a job for sub-benchmarks.

Reply:
i think i've addressed all the benchmark issues.